shared/message/motion/KinematicsModel.proto

```diff
@@ -102,6 +114,7 @@ message KinematicsModel {
         mat3 ankle_block = 8;
         mat3 foot        = 9;
     }
+    // Instantiations of the config containers
```

Comment:
Should this be a line lower?

Reply:
I was doing comments which weren't `@brief` ones, which also described multiple message fields as `//` instead of `///` and with a line of whitespace, to indicate that it wasn't tightly coupled with the next immediate field. Am I overthinking this? what do you think of that?

Reply:
Hm it doesn't feel very intuitive and the extra line looks out of place, it looks like its attached to the stuff above. Two slashes feels like it should cover less lines than three. I'm not sure of a solution, but I think move this down one line.

Reply:
I'll move it one lower. I think that works reasonably well.